samples/cpp/text_generation/README.md

```diff
@@ -11,25 +11,49 @@ There are also Jupyter notebooks for some samples. You can find links to them in
 4. [Support and Contribution](#support-and-contribution)
 
 ## Download and convert the model and tokenizers
-
 The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
-
-It's not required to install [../../export-requirements.txt](../../export-requirements.txt) for deployment if the model has already been exported.
-
+Install [../../export-requirements.txt](../../export-requirements.txt) if model conversion is required.
 ```sh
-pip install --upgrade-strategy eager -r ../../requirements.txt
+pip install --upgrade-strategy eager -r ../../export-requirements.txt
 optimim-cli export openvino --model <model> <output_folder>
 ```
+If a HF model is already converted (as example [OpenVINO/TinyLlama-1.1B-Chat-v1.0-int8-ov](https://huggingface.co/OpenVINO/TinyLlama-1.1B-Chat-v1.0-int8-ov)), it can be download directly via huggingface-cli
+```sh
+huggingface-cli download <model> --local-dir <output_folder>
```

Comment:
it's installed with export-requirements.txt

![{3DF6614F-7FC0-4A07-A563-3BF1E166EEC7}](https://github.com/user-attachments/assets/8026b670-658c-4ddb-ac8c-961f9947e96a)

Reply:
fixed